code/modules/client/preferences_savefile.dm

```diff
@@ -80,6 +80,9 @@
 		sound_toggles |= (SOUND_ADMIN_MEME|SOUND_ADMIN_ATMOSPHERIC)
 		S["toggles_sound"] << sound_toggles
 
+	if(savefile_version < 21)
+		S["grade_path"] << "ME7"
```

Comment:
```suggestion
		S["sea_path"] << "ME7"
```
I have no idea how savefiles work, but this is the name we seem to be using? Can probably rename it to `sea_paygrade` for the sake of clarity anyway.

Reply:
```suggestion
		S["sea_path"] << "Gunnery Sergeant"
```
And this is part of a spread around suggestion for readability, which is certainly very readable itself and not confusing in the least, why do you ask.